README.md

```diff
@@ -12,9 +12,7 @@ lsp-mode client leveraging [Pyright language server](https://github.com/microsof
 ``` emacs-lisp
 (use-package lsp-pyright
   :ensure t
-  :hook (python-mode . (lambda ()
-                          (require 'lsp-pyright)
```

Comment:
If `(require 'lsp-pyright)` is removed, the client may not be registered correctly.